src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline

```diff
@@ -431,15 +431,16 @@ module Library
       #   require "library/v1"
       #
       #   library_service_client = Library::V1::LibraryServiceClient.new
-      #   shelf = Google::Example::Library::V1::Shelf.new
+      #   shelf = {}
```

Comment:
I think it might be more helpful to users if we retain the full type name when it's empty. WDYT?